pkg/convert/arb.go

```diff
@@ -141,11 +142,32 @@ func ToArb(translation model.Translation) ([]byte, error) {
 	// dst length = number of messages + number of potential descriptions (same as number of messages) + locale.
 	dst := make(map[string]interface{}, len(translation.Messages)*2+1)
 
+	patternsToMsg := func(patterns []ast.Pattern) string {
+		var text string
+
+		for _, p := range patterns {
+			if textPattern, ok := p.(ast.TextPattern); ok {
+				text += string(textPattern)
+			}
+		}
+
+		return text
+	}
+
 	// "und" (Undetermined) language.Tag is also valid BCP47 tag.
 	dst["@@locale"] = translation.Language
 
 	for _, msg := range translation.Messages {
-		dst[msg.ID] = "" // TODO: convert from MF2 format.
+		tree, err := ast.Parse(msg.Message)
+		if err != nil {
+			return nil, fmt.Errorf("parse mf2 message: %w", err)
+		}
+
+		if simpleMessage, ok := tree.Message.(ast.SimpleMessage); ok {
+			msg.Message = patternsToMsg(simpleMessage.Patterns)
+		}
+
+		dst[msg.ID] = msg.Message
```

Comment:
```suggestion
		if simpleMessage, ok := tree.Message.(ast.SimpleMessage); ok {
			dst[msg.ID] = patternsToMsg(simpleMessage.Patterns)
		}

returns errors.New(...) // complex is not implemented yet
```

Reply:
This would be better
```suggestion
		switch mf2Msg := tree.Message.(type) {
		case nil:
			dst[msg.ID] = ""
		case ast.SimpleMessage:
			dst[msg.ID] = patternsToMsg(mf2Msg.Patterns)
		case ast.ComplexMessage:
			return nil, fmt.Errorf("complex message not supported")
		}
```